### PR TITLE
Added checkbox toggle when click on product-card

### PR DESCRIPTION
**Pull request by tiftaylor**
- Added a quick gitignore bit for my mac auto making DS_store files
- Made it so product-cards can toggle the checkbox input on single item cards
- Disabled checkbox action when clicking on Details word/Carrot icon near the word
- Added code to check if a checkbox card or multi-select card so the code only runs on check-box cards

Asana Task: https://app.asana.com/0/1157808567895197/1175032686658026/f

### CODE REVIEW

File: hifireg/registration/static/scripts/ticket_selection.js [hunk 1]
```diff
@@ -1,3 +1,29 @@
+//product-card clicks can togggle checkbox
+$('.product-card').on('click', function(e) {
+  //if this is not a checkbox card, move on
+  var checkBox = $(this).find('.product-checkbox');
+  if (checkBox.length === 0) {
+    return;
+  }
+
+  //if checkbox is on card, grab the first input element
```
Comment (blueridger) [inline hifireg/registration/static/scripts/ticket_selection.js]: This comment doesn't really help me understand the purpose of the following block, which is to ignore clicks on the checkbox itself.

File: hifireg/registration/static/scripts/ticket_selection.js [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+  //Making no action for checkbox when user clicks "Details" carrot icon
+  if (e.target.tagName.toLowerCase() === 'details') {
+    return;
+  }
+  
+  //if you click on the card outside the checkbox, 
+  //toggle the checkbox and trigger change event
+  var isChecked = checkBox.prop('checked');
+  checkBox.prop('checked', !isChecked);
+  checkBox.change();
```
Comment (blueridger) [inline hifireg/registration/static/scripts/ticket_selection.js]: I think these three lines could be replaced with `checkBox.click()` Try it out?

**Review by blueridger — CHANGES_REQUESTED** [review]
Awesome job overall. A worthy solution. Some minor comments for your consideration.
Also, consider squashing your commits. Check out `git rebase -i` and `git commit --amend`

File: hifireg/registration/static/scripts/ticket_selection.js (hunk 1 above)
Comment (tiftaylor) [reply hifireg/registration/static/scripts/ticket_selection.js]: Changed wording

File: hifireg/registration/static/scripts/ticket_selection.js (hunk 2 above)
Comment (tiftaylor) [reply hifireg/registration/static/scripts/ticket_selection.js]: code replaced

**Review by blueridger — APPROVED** [review]

**Pull request merged**